app/core/src/main/java/com/fsck/k9/message/Attachment.kt

```diff
@@ -6,6 +6,7 @@ interface Attachment {
     val contentType: String?
     val name: String?
     val size: Long?
+    val internalAttachment: Boolean
```

Comment:
The API feels nicer if we change this to `isInternalAttachment`.

Reply:
Done. I didn't realise Kotlin took into account the whole JavaBean `boolean`/`Boolean` quagmire.